apps/allocations/app/components/Panel/NewBudget.js

```diff
@@ -0,0 +1,116 @@
+import PropTypes from 'prop-types'
+import React from 'react'
+import { Info, TextInput } from '@aragon/ui'
+import { BigNumber } from 'bignumber.js'
+
+import { Form, FormField } from '../Form'
+import { isStringEmpty } from '../../utils/helpers'
+
+// TODO:: This should be votingTokens from account?
+const INITIAL_STATE = {
+  name: '',
+  nameError: true,
+  amount: '',
+  amountError: true,
+}
+
+class NewBudget extends React.Component {
+  static propTypes = {
+    onCreateBudget: PropTypes.func.isRequired
+  }
+
+  state =  INITIAL_STATE
+
+  changeField = e => {
+    if (e.target.name === 'name') {
+      if (isStringEmpty(e.target.value)) {
+        this.setState({ nameError: true })
+      }
+      else {
+        this.setState({ nameError: false })
+      }
+    }
+    else if (e.target.name === 'amount') {
+      if (isStringEmpty(e.target.value) || BigNumber(e.target.value).isNaN()) {
+        this.setState({ amountError: true })
+      }
+      else {
+        this.setState({ amountError: false })
+      }
+    }
+    this.setState({ [e.target.name]: e.target.value })
+  }
+
+  createBudget = () => {
+    const { name, amount } = this.state
+
+    this.props.onCreateBudget({
+      name,
+      amount,
+    })
+    this.setState(INITIAL_STATE)
+  }
+
+  render() {
+
+    return (
+      <React.Fragment>
+
+
+        <Form
+          onSubmit={this.createBudget}
+          // heading={this.props.heading}
```

Comment:
But are you aware that it wasn't me who added that? It's green only because I renamed the file. Should I still remove it? It seems a little harsh to remove other people's comments without really knowing why they're there, no? Like, I never put comments and I don't think it's a good practice, but removing them like that?

Reply:
I think is safe to remove if the heading is not in the designs and the prop is not passed, but is not a big deal, so let's keep for it for now and perform some cleanup during the final checks

Reply:
git doesn't seem to help us with the diff here, which puts me in a "let's clean this stuff up while we're here" mood. If git understood that this was a rename, and not a brand new file, I'd agree with you Emilio.

Also, I don't think it's harsh to clean up code. 😃